include/grub/i386/linux.h

```diff
@@ -154,11 +154,11 @@ struct linux_i386_kernel_header
 struct linux_kernel_info
 {
   grub_uint32_t header;
-  grub_uint32_t size;
-  grub_uint32_t size_total;
+  grub_uint32_t size;		/* In bytes, excluding var_len_data[] */
+  grub_uint32_t size_total;	/* In bytes, including var_len_data[] */
```

Comment:
Are these redundant with each other, or could the offset of `var_len_data` change in the future?

Reply:
That’s fine, consider this resolved.